docs/recipes/installTips.md

```diff
@@ -364,6 +363,254 @@ GUI will open!
 
 Note: Based on issues  #1380 and #2011, thanks!
 
+## Installation on Ubuntu 22.04.1 LTS (Jammy Jellyfish)
+
+Hello! Another cookbook entry on how to install your freshly installed 22.04 LTS system for DLC use. Namely, CUDA, drivers, Docker, and anaconda!
+
+### Let's start with CUDA support for your GPU:
+Get update:
+
+`sudo apt update`
+
+Install git (distributed version control system):
+
+`sudo apt install git`
+
+Install gcc (compiler):
+
+`sudo apt install gcc`
+
+Install CUDA:
+
+```python
+
+wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/cuda-ubuntu2004.pin
```

Comment:
Shouldn't this be
```suggestion
wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/x86_64/cuda-ubuntu2204.pin
```
as the drivers are being installed are for Ubuntu 22.04?

Note that if this changes, a lot of the versions below need to be updated as well